test/integration/util/util.go

```diff
@@ -225,6 +225,25 @@ func AddTaintToNode(cs clientset.Interface, nodeName string, taint v1.Taint) err
 	return err
 }
 
+// RemoveTaintOffNode removes taints off a specific node
+func RemoveTaintOffNode(cs clientset.Interface, nodeName string, taint v1.Taint) error {
+	node, err := cs.CoreV1().Nodes().Get(context.TODO(), nodeName, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+	copy := node.DeepCopy()
```

Comment:
i see this copy is following the existing code in:
https://github.com/kubernetes/kubernetes/blob/d2c82dd40c06d8fa405e43813bd00e99fc5bbc70/test/integration/util/util.go#L217
so you can leave this as is.

but i wanted to note that, i don't think a copy is needed because Get() should be returning a newly constructed *Node object. modifying it and passing the same object to Update() should work fine.

Reply:
Echo @neolit123's suggestion, basically if the obj is from API server, we don't need to do DeepCopy. But a bit embarrassingly, a lot of places are deepCopying everything... it may get you confused.

The place needs to really do deepCopy is getting API object from SharedLister. In which case, the object pointer is shared across all interested parties.

Reply:
Thanks both for this suggestion, I have added a separate commit where I changed both `AddTaintToNode` and `RemoveTaintOffNode` so that they are both consistent and unnecessarily copy the node object.

However, I can easily remove the commit if you think this is out of scope for this PR.